src/constants/dicom/uids/TransferSyntaxUID.js

```diff
@@ -0,0 +1,6 @@
+module.exports = Object.freeze({
+    IMPLICIT_LITTLE_ENDIAN: "1.2.840.10008.1.2",
+    EXPLICIT_LITTLE_ENDIAN: "1.2.840.10008.1.2.1",
+    DEFLATED_EXPLICIT_LITTLE_ENDIAN: "1.2.840.10008.1.2.1.99",
+    EXPLICIT_BIG_ENDIAN: "1.2.840.10008.1.2.2"
```

Comment:
These are unfortunately defined multiple places in the code already.  It does make sense to centralize them and use them consistently everywhere. 

Another stylistic thing, I think it would be easier to have a smaller number of files, so maybe just one dicomConstants.js for anything that's not proper to the data dictionary.

Reply:
I moved such items to one file ./src/constants/dicom.js. 

Having it in a separate folder is a bit personal preference of me, but in our own projects, we make more use of constants and I have more dedicated files. Would keep such decision up to you. I can also move it to the ./src folder as dicomConstants.js if you like that more.

Reply:
Yes, I like this form just fine.  Makes more sense to me.